event_app/settings.py

```diff
@@ -1,150 +1,13 @@
-from .local_settings import *
```

Comment:
Isn't it a bit overkill to break settings module so heavily ?

Reply:
I find it to be more organized and clean :) Usually, after 1 year of development or so, the settings.py gets really cluttered.